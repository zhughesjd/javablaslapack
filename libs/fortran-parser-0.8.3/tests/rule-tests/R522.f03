! Testing bind-stmt, R522
10  bind(c) a
BIND(C) :: B, C, /common_block_name/
end
