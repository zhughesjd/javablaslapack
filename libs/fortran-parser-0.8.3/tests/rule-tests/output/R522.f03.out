R1101:main-program__begin:
R 509:language-binding-spec: language=[@7,35:35='c',<206>,2:9] hasName=false
R 523:bind-entity-list__begin:
R 523:bind-entity: entity=[@10,38:38='a',<206>,2:12] isCommonBlockName=false
R 523:bind-entity-list: count=1
R 522:bind-stmt: label=[@2,26:27='10',<10>,2:0]
R 509:language-binding-spec: language=[@14,45:45='C',<206>,3:5] hasName=false
R 523:bind-entity-list__begin:
R 523:bind-entity: entity=[@19,51:51='B',<206>,3:11] isCommonBlockName=false
R 523:bind-entity: entity=[@22,54:54='C',<206>,3:14] isCommonBlockName=false
R 523:bind-entity: entity=[@26,58:74='common_block_name',<206>,3:18] isCommonBlockName=true
R 523:bind-entity-list: count=3
R 522:bind-stmt: label=null
R 204:specification-part: numUseStmts=0 numImportStmts=0 numDeclConstructs=2
R1103:end-program-stmt: id=null
R1101:main-program: hasProgramStmt=false hasExecutionPart=false hasInternalSubprogramPart=false
